nemo/export/vllm/model_config.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         self.disable_sliding_window = disable_sliding_window
         self.served_model_name = nemo_checkpoint
         self.multimodal_config = None
+        self.mm_processor_kwargs = {}
```

Comment:
This is needed for vllm >= 0.6.2, see https://github.com/vllm-project/vllm/blob/v0.6.2/vllm/engine/llm_engine.py#L277